src/cargo/util/toml/mod.rs

```diff
@@ -1284,18 +1276,18 @@ fn to_real_manifest(
         }
     }
 
-    verify_lints(
-        resolved_toml.resolved_lints().expect("previously resolved"),
-        gctx,
-        warnings,
-    )?;
-    let default = manifest::TomlLints::default();
-    let rustflags = lints_to_rustflags(
-        resolved_toml
-            .resolved_lints()
-            .expect("previously resolved")
-            .unwrap_or(&default),
-    );
+    let resolved_lints = original_toml
```

Comment:
Use of `original_toml` should be for exception cases in `to_real_manifest`

Reply:
Whoops, I must've forgotten to switch that when I was moving this to here